service/templates/projects.html

```diff
@@ -267,6 +268,15 @@ <h2> Settings for [[displayName]]</h2>
           </div>
         </div>
 
+        <div id="logging-info">
+          <div class="h3">Default Logger</div>
+          <div style="left:-15px" class="col-lg-4">
+            <select name="repeatSelect" id="repeatSelect" ng-model="settingsFormData.default_logger">
+              <option ng-repeat="logger in validDefaultLoggers">[[logger]]</option>
```

Comment:
"" seems to be a valid option as well, but it isn't in this list

Reply:
how do I add "" to the list?